src/main/kotlin/org/neo4j/graphql/Cypher.kt

```diff
@@ -0,0 +1,39 @@
+package org.neo4j.graphql
+
+import graphql.schema.GraphQLFieldDefinition
+import graphql.schema.GraphQLObjectType
+import graphql.schema.GraphQLType
+
+data class Cypher(val query: String, val params: Map<String, Any?> = emptyMap()) {
+    fun with(p: Map<String, Any?>) = this.copy(params = this.params + p)
+    fun escapedQuery() = query.replace("\"", "\\\"").replace("'", "\\'")
+
+    companion object {
```

Comment:
I would move these somewhere else don't really belong into this data class imho.
It looks almost a bit like the DSL bits we discussed.

Reply:
already removed in the 2nd commit, the 1st commit is just to have an idea about the refactoring